src/Visitor/Php/Symfony/FormTypeChoices.php

```diff
@@ -75,21 +75,31 @@ public function enterNode(Node $node)
             if (!$item->key instanceof Node\Scalar\String_) {
                 continue;
             }
+            $domain = null;
```

Comment:
I think this $domain variable initialization should be moved before the `foreach ($node->items as $item) {` line as my tests have shown that this domain is overwritten (except of scenario when the choice_translation_domain is very last option in ChoiceType configuration array.

Reply:
Thank you